contracts/token/T.sol

```diff
@@ -3,10 +3,11 @@
 pragma solidity 0.8.4;
 
 import "@thesis/solidity-contracts/contracts/token/ERC20WithPermit.sol";
+import "@thesis/solidity-contracts/contracts/token/MisfundRecovery.sol";
 
 /// @title T token
 /// @notice Threshold Network T token.
-contract T is ERC20WithPermit {
+contract T is ERC20WithPermit, MisfundRecovery {
```

Comment:
Can we add a simple happy-path test case to check that misfund recovery is possible for `T` contract? I know it's an external contract tested elsewhere but we should check that `T` contract gives a basic functionality expected after extending the `MisfundRecovery` contract.